src/main/kotlin/no/nav/syfo/infrastructure/kafka/BehandlerMeldingProducer.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.syfo.infrastructure.kafka
+
+import no.nav.syfo.application.IBehandlerMeldingProducer
+import no.nav.syfo.domain.BehandlerMelding
+import no.nav.syfo.domain.Personident
+import no.nav.syfo.domain.serialize
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class BehandlerMeldingProducer(private val produder: KafkaProducer<String, BehandlerMeldingRecord>) :
+    IBehandlerMeldingProducer {
+
+    override fun send(
+        personident: Personident,
+        behandlermelding: BehandlerMelding,
+        behandlermeldingPdf: ByteArray
+    ): Result<BehandlerMelding> =
```

Comment:
Kanskje runCatching passer bra her?

Reply:
I stedet for try-catch tenker du? 😊 Hvis vi vil logge at noe gikk feil i produsenten må vi i så fall logge det på et høyere nivå.

Reply:
Rydder opp i neste PR.